crates/model/src/solver_competition.rs

```diff
@@ -77,6 +77,9 @@ pub struct CompetitionAuction {
 pub struct SolverSettlement {
     pub solver: String,
     pub objective: Objective,
+    #[serde(flatten)]
+    pub score: Score, // auction based score
+    pub ranking: usize, // auction based ranking
```

Comment:
At first I thought this was redundant. Could you add to the documentation that this is temporarily needed as the scored settlements are ordered by objective value ATM and this represents how they would be ranked after switching to the new scoring scheme?